core/blockchain.go

```diff
@@ -2336,24 +2336,25 @@ func (bc *BlockChain) UpdateValidatorVotingPower(
 				return nil, err
 			}
 
-			aprComputed, err := apr.ComputeForValidator(
-				bc, block, wrapper,
-			)
-			if err == nil && aprComputed != nil {
-				a := *aprComputed
+			stats.APR = numeric.ZeroDec()
+			if wrapper.TotalDelegation().Cmp(common.Big0) > 0 {
+				aprComputed, err := apr.ComputeForValidator(
+					bc, block, wrapper,
+				)
+				if err != nil {
+					if err == apr.ErrInsufficientEpoch {
+						utils.Logger().Info().Msg("insufficient past epochs to compute apr")
+					} else {
+						return nil, err
+					}
+				}
 				utils.Logger().Info().
```

Comment:
please remove this whole call, its not adding useful info because you don't know to whom the apr is computed to, so its just writing that it computed apr but we can't debug and say to whom, so its not useful.